### PR TITLE
Issue 1338/add event link

### LINKED ISSUE #1338: Add link to event to CalendarDayView
## Description
In calendar day view the event should be clickable 

## Steps to reproduce
1. Go to http://localhost:3000/organize/1/projects/calendar?focusDate=2023-05-11&timeScale=day)
2. Click on any event title

### Expected Behaviour
Go to Event page

### Actual Behaviour
Nothing happens, title is not clickable

### PR DESCRIPTION
**Pull request by rebecarubio**
## Description
This PR adds the event link in Event Calendar Day view, and fixes the color of event title and participants icon.

## Screenshots
https://github.com/zetkin/app.zetkin.org/assets/36491300/8a6aa9cd-22d6-4246-8c31-f57887c36253

## Changes
* Adds NexLint and Link component to wrap the Element component and make it clickable
* Changes the color of the Participants icon to dark grey
* Changes the color of Event title to dark grey

## Notes to reviewer
None

## Related issues
Resolves #1338

### CODE REVIEW

File: src/features/calendar/components/CalendarDayView/Day/Event.tsx [hunk 1]
```diff
@@ -38,75 +39,96 @@ const Event = ({ event }: { event: ZetkinEvent }) => {
...
-    <Box
-      display="flex"
-      flexDirection="row"
-      justifyContent="space-between"
-      padding={2}
-      sx={{ backgroundColor: 'white', borderRadius: '3px' }}
-      width="100%"
+    <NextLink
+      href={`/organize/${event.organization.id}/${
+        event.campaign ? `projects/${event.campaign.id}` : 'standalone'
```
Comment (richardolsson) [inline src/features/calendar/components/CalendarDayView/Day/Event.tsx:44]: This link doesn't look correct to me. When there is no campaign, the output will be incorrect. It should be `/organize/ID/projects/standalone/events/ID` but just becomes `/organize/ID/standalone/events/ID` (missing `projects`), am I right?

**Review by richardolsson — CHANGES_REQUESTED** [review]
Works good for events in projects, but I don't think it will work for standalone events?

File: src/features/calendar/components/CalendarDayView/Day/Event.tsx (hunk 1 above)
Comment (rebecarubio) [reply src/features/calendar/components/CalendarDayView/Day/Event.tsx:44]: You're right! fixing it!

**Review by richardolsson — APPROVED** [review]
This looks good to me now and seems to be working as expected. Feel free to merge if tests pass! 🚀

**Pull request merged**